VideoLocker/res/layout/fragment_handout.xml

```diff
@@ -1,27 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/grey_act_background"
-    android:orientation="vertical" >
 
-    <org.edx.mobile.view.custom.EdxWebView
-        android:id="@+id/webview"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:background="@color/grey_act_background" />
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+            xmlns:tools="http://schemas.android.com/tools"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@color/edx_grayscale_neutral_white"
+            tools:context="org.edx.mobile.view.CourseHandoutActivity">
 
-    <TextView
-        android:id="@+id/downloadMessage"
-        style="@style/downloading_message"
-        android:visibility="gone"
-        android:layout_alignParentTop="true"/>
+    <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:orientation="vertical"
+                    android:paddingEnd="@dimen/common_course_display_margin_side"
+                    android:paddingLeft="@dimen/common_course_display_margin_side"
+                    android:paddingTop="@dimen/common_course_display_margin_side"
+                    android:paddingBottom="@dimen/common_course_display_margin_side"
+                    android:paddingRight="@dimen/common_course_display_margin_side"
+                    android:paddingStart="@dimen/common_course_display_margin_side"
```

Comment:
nit: you don't need to define start and end when they are equal. In fact, all of these are equal you could just use `padding="@dimen/common_course_display_margin_side"` and drop the rest